core/domain/exp_jobs_one_off_test.py

```diff
@@ -668,11 +978,390 @@ def test_migration_job_skips_deleted_explorations(self):
         with self.assertRaisesRegexp(Exception, 'Entity .* not found'):
             exp_services.get_exploration_by_id(self.NEW_EXP_ID)
 
+    def test_exploration_migration_job_produces_no_output(self):
+        """Test that Exploration Migration job produces no output."""
+        exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title='title', category='category')
 
-class ExplorationStateIdMappingJobTest(test_utils.GenericTestBase):
-    """Tests for the ExplorationStateIdMapping one off job."""
+        # Start migration job on sample exploration.
+        job_id = exp_jobs_one_off.ExplorationMigrationJobManager.create_new()
+        exp_jobs_one_off.ExplorationMigrationJobManager.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
 
-    EXP_ID = 'eid'
+        self.assertEqual(
+            exp_jobs_one_off.ExplorationMigrationJobManager.get_output(job_id),
+            []
+        )
+
+
+class InteractionAuditOneOffJobTests(test_utils.GenericTestBase):
+
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    VALID_EXP_ID = 'exp_id0'
+    NEW_EXP_ID = 'exp_id1'
+    EXP_TITLE = 'title'
+
+    def setUp(self):
+        super(InteractionAuditOneOffJobTests, self).setUp()
+
+        # Setup user who will own the test explorations.
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.process_and_flush_pending_tasks()
+
+    def test_exp_state_pairs_are_produced_for_all_interactions(self):
+        """Checks (exp, state) pairs are produced for all interactions."""
+        exploration = exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title='title', category='category')
+
+        exploration.add_states(['End'])
+        intro_state = exploration.states['Introduction']
+        end_state = exploration.states['End']
+
+        intro_state.update_interaction_id('TextInput')
+        end_state.update_interaction_id('EndExploration')
+        end_state.update_interaction_default_outcome(None)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start InteractionAuditOneOff job on sample exploration.
+        job_id = exp_jobs_one_off.InteractionAuditOneOffJob.create_new()
+        exp_jobs_one_off.InteractionAuditOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = exp_jobs_one_off.InteractionAuditOneOffJob.get_output(job_id) # pylint: disable=line-too-long
```

Comment:
This can be broken up right? In general, please avoid using pylint pragmas unless it is absolutely necessary.
```suggestion
        actual_output = (
            exp_jobs_one_off.InteractionAuditOneOffJob.get_output(
                job_id)) # pylint: disable=line-too-long
```

Reply:
Done.